package.json

```diff
@@ -9,10 +9,10 @@
     "tdd": "NODE_ENV=unittest jest --watch",
```

Comment:
You need to remove all the dependencies now provided by razzle (webpack, babel, etc)